warehouse/query-core/src/main/java/datawave/query/tables/ScannerSession.java

```diff
@@ -28,12 +20,9 @@
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.log4j.Logger;
 
-import com.google.common.base.Preconditions;
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Queues;
-import com.google.common.util.concurrent.AbstractExecutionThreadService;
-import com.google.common.util.concurrent.MoreExecutors;
+import java.util.*;
```

Comment:
More star imports, probably a setting in your IDE

Reply:
Fixed and changed the setting in intellij so it won't happen again.